app/views/spotlight/custom_fields/form_group/_text.html.erb

```diff
@@ -1,14 +1,23 @@
-<div class="form-group">
+<% inline = local_assigns[:inline].presence %>
+<% label_col = f&.label_col %>
+<% control_col = f&.control_col %>
+<% offset_col = label_col.sub('col', 'offset') %>
```

Comment:
Not super happy with having to do this here.  Happy to entertain additional ideas.